guide/source/software_developer/introduction.rst

```diff
@@ -31,13 +31,13 @@ developers are there. They have a lot of experience in getting HPXML working and
 can help you avoid costly pitfalls.
 
 To join the working group contact `Julie Caracino
-<Julie.Caracino@nyserda.ny.gov>`_.
+<Julie.Caracino@nyserda.ny.gov>`_.[NEED GENERIC EMAIL HERE. NOT PERSON]
 
 GitHub
 ******
 
 The HPXML schemas, example files, and this guide are all `maintained on GitHub
-<https://github.com/hpxmlwg/hpxml>`_. You will always be able to find the latest
+<https://github.com/hpxmlwg/hpxml>`_. [WHO PAYS FOR THIS? WHO WILL ALWAYS PAY FOR THIS? I'M JUST ASKING.  DON'T INCLUDE IN DOC]You will always be able to find the latest
```

Comment:
I'll answer this one now because it's easy: nobody pays for this. GitHub is free to use in they way we're using it and will likely continue to be so for some time. If that changes we can re-evaluate.